### ISSUE TITLE
"enabled" setting overwrites "notify" setting depending on the order

### ISSUE THREAD

**Issue by fab-soundcloud** [anthavio/airbrake-logback#1]
Given this appender config: 

```
<appender name="AIRBRAKE" class="net.anthavio.airbrake.AirbrakeLogbackAppender">
        <apiKey>${AIRBRAKE_API_KEY:-}</apiKey>
        <env>${AIRBRAKE_ENV:-test}</env>
        <notify>ALL</notify>
        <enabled>true</enabled>

        <filter class="ch.qos.logback.classic.filter.ThresholdFilter">
            <level>ERROR</level>
        </filter>
    </appender>
```

As the "enabled" setting actually just changes the "notify" setting by overwriting the log level it does not matter which "notify" level I specify I will be overriden by "EXCEPTION" if I set "enabled" to true although I specified "ALL"

The same happens the other way around. Given this:

```
<appender name="AIRBRAKE" class="net.anthavio.airbrake.AirbrakeLogbackAppender">
        <apiKey>${AIRBRAKE_API_KEY:-}</apiKey>
        <env>${AIRBRAKE_ENV:-test}</env>
        <enabled>false</enabled>
        <notify>ALL</notify>

        <filter class="ch.qos.logback.classic.filter.ThresholdFilter">
            <level>ERROR</level>
        </filter>
    </appender>
```

The notifier will be activated and try to send stuff although I specified "false" in enabled because "notify" ALL turns it on again.

As a workaround we only use "notify" which works as expected but to avoid confusion I would either get rid of the "enabled" setting or fix it to be a boolean and to be independent of the set log level.

**Comment by anthavio** [comment]
I guess "enabled" should take priority over "notify"

**Comment by fab-soundcloud** [comment]
I agree but this is not the case. "Notify" takes the priority if its set after "enabled". The order is important here. I think the notification level and the fact that something is enabled are two different concerns, meaning I guess we need a boolean enabled flag in the implementation?

**Comment by anthavio** [comment]
enabled was implemented as just another way of setting Notify.EXCEPTION" (true) or Notify.OFF (false)
I made it flag on it's own. Released as version 1.0.1

**Comment by fab-soundcloud** [comment]
Cool! Thanks!
